user-interface/src/main/java/life/qbic/datamanager/views/projects/project/experiments/experiment/ExperimentalGroupInput.java

```diff
@@ -47,29 +48,32 @@ public class ExperimentalGroupInput extends CustomField<ExperimentalGroupBean> {
       VariableValueFormatter.format(it.experimentalValue()));
 
   private final List<ComponentEventListener<RemoveEvent>> removeEventListeners;
+  private final TextField nameField;
   private final MultiSelectComboBox<VariableLevel> variableLevelSelect;
   private final NumberField replicateCountField;
   int variableCount = 0;
   private final List<Binder<?>> binders = new ArrayList<>();
 
   /**
    * ExperimentalGroupInput is a {@link CustomField} which contains a {@link MultiSelectComboBox}
-   * allowing the user to define the {@link Condition} and a {@link NumberField} enabling the user
-   * to define the number of {@link BiologicalReplicate} within an {@link ExperimentalGroup}
+   * allowing the user to define the {@link Condition}, a {@link NumberField} enabling the user
+   * to define the number of {@link BiologicalReplicate} within an {@link ExperimentalGroup}, and
+   * a {@link TextField} to optionally name the group.
```

Comment:
NIT: The ordering of the explanation should follow the ordering of the input with the namefield as first entry